src/Composer/Installer.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerPlugin\Composer;
+
+use Composer\IO\IOInterface;
+use Composer\Repository\RepositoryInterface;
+use Contao\ManagerPlugin\Dependency\DependencyResolverTrait;
+use Contao\ManagerPlugin\Dependency\DependentPluginInterface;
+
+class Installer
+{
+    use DependencyResolverTrait;
+
+    /**
+     * @var string
+     */
+    private static $generatedClassTemplate = <<<'PHP'
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerPlugin;
+
+use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Config\ConfigPluginInterface;
+use Contao\ManagerPlugin\Config\ExtensionPluginInterface;
+use Contao\ManagerPlugin\Routing\RoutingPluginInterface;
+
+/**
+ * This class has been auto-generated. It will be overwritten at every run of
+ * `composer install` or `composer update`.
+ * 
+ * @see \Contao\ManagerPlugin\Composer\Installer
+ */
+%s
+{
+    public const BUNDLE_PLUGINS = BundlePluginInterface::class;
+    public const CONFIG_PLUGINS = ConfigPluginInterface::class;
+    public const EXTENSION_PLUGINS = ExtensionPluginInterface::class;
+    public const ROUTING_PLUGINS = RoutingPluginInterface::class;
+
+    /**
+     * @var array
+     */
+    private $plugins;
+
+    /**
+     * @var array
+     */
+    private $disabled = [];
+
+    /**
+     * @param string|null $installedJson
+     * @param array       $plugins
+     */
+    public function __construct($installedJson = null, array $plugins = null)
```

Comment:
This needs to be `string $installedJson` (or maybe `?string $installedJson`).

Reply:
Fixed in bfc084025e12240524c7841e068d5128cdf4081b